### PR TITLE
[bitnami/minio] Add missing features

### PR DESCRIPTION
**Pull request by juan131**
Signed-off-by: juan131 <juanariza@vmware.com>

**Description of the change**

This PR adds several missing features in the chart:

- Multidrive support (distributed mode only)
- Multizone support (distributed mode only)
- PodDisruptionBudget
- Metrics integration with Prometheus Operator
- Loading env from existing ConfigMaps/Secrets
- Defining custom command/args
- Defining custom container port
- Defining custom probes (+ startupProbe)
- Defining custom init containers and sidecars
- Defining custom volumes and mount points

**Benefits**

- Suitable for production environments
- Flexibility

**Possible drawbacks**

None

**Applicable issues**

None

**Additional information**

N/A

**Checklist** 

- [x] Chart version bumped in `Chart.yaml` according to [semver](http://semver.org/).
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[bitnami/chart]`)

### CODE REVIEW

File: bitnami/minio/README.md [hunk 1]
```diff
@@ -189,11 +249,25 @@ Bitnami will release a new chart updating its containers if a new version of the
 
 ### Distributed mode
 
-You can start the MinIO&reg; chart in distributed mode with the following parameter: `mode=distributed`
+This chart provisions a MinIO&reg; server in standalone mode, by default. You can start MinIO&reg; server in [distributed mode](https://docs.minio.io/docs/distributed-minio-quickstart-guide) with the following parameter: `mode=distributed`
```
Comment (rafariossaa) [inline bitnami/minio/README.md:252]: I would rephrase as:
'By default, this chart ...'

**Review by rafariossaa — CHANGES_REQUESTED** [review]
Hi,
  Could you take a look to my suguestions ?

**Comment by Mauraza** [comment]
Thanks for the PR! LGTM!

**Review by rafariossaa — APPROVED** [review]
LGTM

**Comment by lathspell** [comment]
@juan131 : Now I get the following error although I have `--set global.minio.accessKey=` set. Maybe "minio.secretKey" as defined in `_helpers.tpl` should be used? Why are there two variables anyway?

```
Error: UPGRADE FAILED: template: minio/templates/NOTES.txt:82:4: executing "minio/templates/NOTES.txt" at <include "common.errors.upgrade.passwords.empty" (dict "validationErrors" (list $requiredMinioPasswordErrors) "context" $)>: error calling include: template: minio/charts/common/templates/_errors.tpl:18:48: executing "common.errors.upgrade.passwords.empty" at <fail>: error calling fail
    'accessKey.password' must not be empty, please add '--set accessKey.password=$ACCESS_KEY' to the command. To get the current value:

...
```
